tests/test_oml/test_models/test_visualisations.py

```diff
@@ -0,0 +1,43 @@
+from typing import Any, Union
+
+import numpy as np
+import pytest
+from PIL import Image
+
+from oml.models import ResnetExtractor, ViTExtractor
+
+IMAGE_SIZE = (224, 224, 3)
+
+
+def get_image_pillow() -> Image.Image:
+    img = np.random.randint(0, 256, size=IMAGE_SIZE, dtype=np.uint8)
+    return Image.fromarray(img)
+
+
+def get_numpy_image() -> np.ndarray:
```

Comment:
If you decided to test several cases, it would be great to check different `dtype`s for input.

Reply:
I don't really want to overcomplicate it now and support different formats. Instead, I've added a docstring where I describe that we want an image with pixel values in the range of ``[0..255]``.

Reply:
If you want [0..255], you could add assert for ranges or np.uint8